script/lib/config.py

```diff
@@ -60,6 +60,8 @@ def get_brave_version():
   version = os.environ.get('npm_config_brave_version') or brave_browser_package()['version']
   return 'v' + version.split('+')[0]
 
+def get_raw_version():
+  return os.environ.get('npm_config_brave_version') or brave_browser_package()['version']
```

Comment:
I think you can simplify `get_brave_version` and `get_chrome_version` to call `get_raw_version` now

Reply:
Unless maybe you mean for `get_raw_version` to just be the same as `get_brave_version` without the 'v'? In which case I think you'd want `version.split('+')[0]`